netty/src/main/java/io/grpc/transport/netty/NettyServerBuilder.java

```diff
@@ -181,7 +206,8 @@ protected ServerEssentials buildEssentials() {
         ? SharedResourceHolder.get(Utils.DEFAULT_WORKER_EVENT_LOOP_GROUP)
         : userWorkerEventLoopGroup;
     NettyServer server = new NettyServer(address, channelType, bossEventLoopGroup,
-        workerEventLoopGroup, sslContext, maxConcurrentCallsPerConnection);
+        workerEventLoopGroup, sslContext, maxConcurrentCallsPerConnection, connectionWindowSize,
```

Comment:
Meh. Although it may be strange, nothing is "invalid" about it. It seems a pointless restriction.